api/report_server.thrift

```diff
@@ -341,20 +377,18 @@ service codeCheckerDBAccess {
                  throws (1: shared.RequestFailed requestError),
 
 
-  // * If (filepath, content_hash) is in the DB return (existing fileId, false).
-  // * If only the content_hash matches, a (new fileId, false) is returned.
-  // * If the filepath matches, but content_hash not, a (new file_id, true) is returned.
-  // * If none of them matches a (new file_id, true) is returned.
-  NeedFileResult needFileContent(
-                                 1: string filepath,
-                                 2: string content_hash)
-                                 throws (1: shared.RequestFailed requestError),
-
-  bool addFileContent(
-                      1: string content_hash,
-                      2: string file_content,
-                      3: optional Encoding encoding)
-                      throws (1: shared.RequestFailed requestError),
+  // Add the necessary file contents. If the file is already stored in the
+  // database then its ID returns. This function must not be called between
```

Comment:
While the description here is correct, it is written in prose too much. Change it to a way that the "what the function does" and "what to pay attention for" is at least separated by an empty line.